net/ip/native_sockets/syscfg.yml

```diff
@@ -0,0 +1,36 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+syscfg.defs:
+    NATIVE_SOCKETS_MAX:
+        description: 'The number of allocated sockets.'
+        value: 8
+    NATIVE_SOCKETS_MAX_UDP:
+        description: 'The maximum UDP datagram size (send and receive).'
+        value: 2048
+    NATIVE_SOCKETS_POLL_ITVL:
+        description: >
+            The frequency at which to poll for received data.  Units
+            are OS ticks.
+        value: 1
```

Comment:
Yikes, I didn't mean to make the default 1!  I have changed it back to OS_TICKS_PER_SEC / 5.